### PR TITLE
Allow user to update his profile

### PR DESCRIPTION
**Pull request by caike08**
Hey buddy,

I'm sending an important enhancement to our system. Now users can update their profiles, as well as include some fields we didn't include in our sign in form. So basically this PR is:
- Users can update their infos
- Layout improvement when a job is applied
- Job status is better displayed now
- LESS fix, due to an recommendation

screenshot from 2016-10-24 20 18 18
screenshot from 2016-10-24 20 18 41

### CODE REVIEW

**Comment by MarceloBorgesP** [comment]
Looks great man! good job :+1:

File: imports/ui/pages/panel/settings/index.js [hunk 1]
```diff
@@ -1,18 +1,387 @@
...
+          profile: {
+            formacao: {
+              formacao: targetValue(target["formacao"]),
+              curso: targetValue(target["curso"]),
+              conclusao: targetValue(target["conclusao"])
+            }
+          }
+        }
+
+        Meteor.call('updateUser', user, formID, FlowRouter.getParam('id'),function(error, result) {
```
Comment (MarceloBorgesP) [inline imports/ui/pages/panel/settings/index.js]: You don't need to repeat this same method call over and over, just call it after the switch case

Comment (caike08) [reply imports/ui/pages/panel/settings/index.js]: It's been fixed! :)

**Comment by MarceloBorgesP** [comment]
Great job man!

**Pull request merged**